build/jobs/provision-deploy.yml

```diff
@@ -22,6 +22,12 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: string
   default: 'tool'
+- name: sqlServerAdminPassword
+  type: string
+  default: 'test@123'
```

Comment:
I don't think we should provide a default password for the SQL server.

Reply:
Just realized that we would need to set default value as provision-deploy.yml is used for cosmosdb site as well. I am setting it to empty value as default which will error out if we don't pass in good value for sql Server